presto-docs/src/main/sphinx/release/release-0.221.rst

```diff
@@ -0,0 +1,46 @@
+=============
+Release 0.221
+=============
+
+General Changes
+---------------
+* Add a variant of :func:`strpos` to return the position of the N-th instance of the substring.
+* Add :func:`strrpos` that returns position of the N-th instance of substring from the back of a string.
+* Add aggregation function :func:`entropy`.
```

Comment:
Should we explain the function `entropy` as well since we have done something similar for `strrpos`

Reply:
@rongrong can you give a brief description?

Reply:
I think it's OK to just link to the documentation.